tt_eager/tt_lib/fallback_ops/fallback_ops.py

```diff
@@ -475,6 +518,7 @@ def __init__(
 
     @convert_tt_tensors_wrapper
     def forward(self, input: ttl_tensor.Tensor) -> ttl_tensor.Tensor:
+        assert False
```

Comment:
Added this for local testing to ensure no convs are run on the fallback path. I will remove this later before running CI tests.